src/components/LoginComponent/LoginComponent.tsx

```diff
@@ -6,7 +6,7 @@ import passThumb from '../../assets/pass-thumb.png';
 import { Auth } from 'aws-amplify';
 
 interface ILoginProps{
-    loginType:string
+    loginType?:string
```

Comment:
Please refrain from turning prop types into nullables just for testing right now. Sending dummy props for tests shold work just as well, especially for "shallow" type tests.